srcpkgs/seatd/template

```diff
@@ -0,0 +1,36 @@
+# Template file	for	'seatd'
+pkgname=seatd
+version=0.2.0
+revision=1
+build_style=meson
+configure_args="-Dexamples=disabled -Dlogind=enabled"
```

Comment:
Make `elogind` a build option, perhaps?

Reply:
Yes, it should be optional and we need a parallel option in wlroots.

Reply:
Is wlroots going to move exclusively to seatd? Would be nice if we could control everything with libseat build options.

Reply:
I that's the long term plan as far as I know, but I wouldn't expect that to happen before the next wlroots release. Agreed that that being able to control everything with libseat build options would be best.